### ISSUE TITLE
MobyLinuxVM failed to start [Hyper-V]

### ISSUE THREAD

**Issue by jophlyn** [docker/for-win#484]
### Expected behavior
Docker running after install
### Actual behavior
'MobyLinuxVM' could not initialize.

'MobyLinuxVM' failed to start.
### Information

  - Diagnostic ID from "Diagnose & Feedback" in the menu.
F6143753-3D13-4B54-B7B3-936C5DEF2646/2017-02-10_09-19-47
  - a reproducible case if this is a bug, Dockerfiles FTW
  - page URL if this is a docs issue or the name of a man page
  - host distribution and version (Windows version, build number, etc)
Version: 1.13.1 (10072)
Channel: Stable
Sha1: 94675c5a765e2c51fba6c10b87e0a1a4ed3dde51
Started on: 2017/02/10 09:18:17.088
Resources: C:\Program Files\Docker\Docker\Resources
OS: Windows 10 Pro
Edition: Professional
Id: 1511
Build: 10586
BuildLabName: 10586.162.amd64fre.th2_release_sec.160223-1728
### Steps to reproduce the behavior
I had a working version of Docker for Windows on my system.  Everything was great until I installed the Remote Server Administration Tools for Windows 10.  Ever since I have seen the same error from Docker whenever I try to run or update it.  I've done updates, uninstalled and reinstalled, and I've long since uninstalled what (I think) was the offending software, to no avail.  I've searched every bug related to this issue and nothing works.

Please help.

**Comment by rneugeba** [comment]
Thanks for your report. Your system seems to have both bits of  VirtualBox and VMware installed. In particular your networking still seems to be running through Virtual Box. Hyper-V does not work with either VirtualBox nor VMare installed (see [here](https://docs.docker.com/docker-for-windows/#what-to-know-before-you-install)).

Out of curiosity could you provide the output of `get-computerinfo *hyper*` from an elevate shell.

Then can you try to uninstall Virtual Box and VMware. You probably need to reboot. And try again. If this doesn't work (sometimes there are issues...), can you try disabling the Hyper-V feature and reboot. And then try starting Docker for Windows again. This will re-enable Hyper-V, which requires yet another reboot unfortunately.

**Comment by jophlyn** [comment]
I tried to run the get-computerinfo command, and it failed.  I then learned that it was a command introduced in Power Shell 5.1, and I only had 5.0 installed.  After digging to figure out why the latest Power Shell wasn't installed, I discovered that because of some change in IT Update policy, my computer hasn't received an update in several months.  A few hours of applying Windows updates later, Docker now works on my computer.  Thank you for your help... n00b out.
